lang/tests/src/test/scala/com/wavesplatform/lang/compiler/ContractCompilerTest.scala

```diff
@@ -706,11 +706,11 @@ class ContractCompilerTest extends PropSpec with PropertyChecks with Matchers wi
         """.stripMargin
       Parser.parseContract(script).get.value
     }
-    compiler.ContractCompiler(ctx, expr, V3) should produce("must be less than")
+    compiler.ContractCompiler(ctx, expr, V3) should produce("exceeds 255")
```

Comment:
Will it be visible in the text of this error how much the limit is exceeded?

Reply:
Fixed